From 96292ce20b75ef27851ba75aed1aafb15a207a79 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Thu, 15 Jan 2015 02:01:57 +0100
Subject: [PATCH 12/13] lib: Bad bugfix for finding  ~/.lircrc

Basically, the common library code did not find ~/.lircrc, just
~/.config/lircrc. Fixed.
---
 lib/lirc_client.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/lirc_client.c b/lib/lirc_client.c
index c441803..df2eaef 100644
--- a/lib/lirc_client.c
+++ b/lib/lirc_client.c
@@ -807,7 +807,7 @@ static char *lirc_getfilename(const char *file, const char *current_file)
 			if (filename == NULL) {
 				return NULL;
 			}
-			strcat(filename, LIRCRC_USER_FILE);
+			strcat(filename, "/" LIRCRC_USER_FILE);
 		}
 		filename = realloc(filename, strlen(filename) + 1);
 	} else if (strncmp(file, "~/", 2) == 0) {
-- 
2.2.2

